src/components/screens/GuideScreen/Hero.js

```diff
@@ -105,6 +105,29 @@ const Languages = styled.div`
 const LanguagesLabel = styled.span`
   font-weight: ${typography.weight.bold};
 `;
+const LanguageLinkStyles = `
+  && {
+    color: ${styles.color.lightest};
+    &:hover {
+      color: ${rgba(styles.color.lightest, 0.5)};
+    }
+    &:visited{
+      color: ${styles.color.lightest};
+    }
+    &:active {
+      color: ${rgba(styles.color.lightest, 1)};
+    }
```

Comment:
I think we should take these custom link styles out and instead just pass the `inverse` prop to `GatsbyLink`:

```javascript
<LanguagesLink inverse
...
```

They are similar enough that it feels like using the design system's prebuilt styles is the right approach. Thoughts?

Reply:
Sounds good to me! I wondered about if this could be better handled with the design system. I took a peek at the code in the design system and wasn't able to find a reference to it, so i implemented the "custom styling", i have t take a deeper look at the code it seems. I'm updating the component accordingly

Reply:
No worries! I figured that was the case. For reference, we have the deployed Storybook for the design system here:

https://storybook-design-system.netlify.com/?path=/docs/*

It is also linked in the [repo](https://github.com/storybookjs/design-system).

Reply:
gonna bookmark it for future reference. Thanks for the links.